constants.py

```diff
@@ -371,6 +371,10 @@
 kTargetRelativeDriveAngleIGain = 0
 kTargetRelativeDriveAngleDGain = 0
 
+kRotationPGain = 1.3
```

Comment:
that is H I G H, even though it's automated idk if that's a good idea. Maybe a bit slower so that people can comprehend what it's doing but we can test it on ppl

Reply:
It needs the high gain to keep up, it’s just a number and without the higher gain it doesn’t actually end on the right rotation